src/components/Registration/BusinessLookup.vue

```diff
@@ -39,56 +40,34 @@
       </v-autocomplete>
     </div>
 
-    <div v-if="state === States.SUMMARY && haveBusiness"
-      class="summary-block d-flex justify-space-between align-center">
+    <div v-if="state === States.SUMMARY && haveBusiness" class="summary-block mt-5">
       <v-row no-gutters>
         <v-col cols="9">
           <v-row no-gutters>
-            <v-col cols="12" sm="3">
-              <label>Name</label>
-            </v-col>
-            <v-col cols="12" sm="9">
+            <v-col cols="12">
+              <label>Name: </label>
               <span>{{businessName}}</span>
             </v-col>
           </v-row>
-          <v-row no-gutters>
-            <v-col cols="12" sm="3">
-              <label>Incorporation Number</label>
-            </v-col>
-            <v-col cols="12" sm="9">
+          <v-row no-gutters class="mt-1">
+            <v-col cols="12">
+              <label>Incorporation Number: </label>
               <span>{{identifier}}</span>
             </v-col>
           </v-row>
-          <v-row no-gutters>
-            <v-col cols="12" sm="3">
-              <label>Business Number</label>
-            </v-col>
-            <v-col cols="12" sm="9">
+          <v-row no-gutters class="mt-1">
+            <v-col cols="12">
+              <label>Business Number: </label>
               <span>{{businessNumber}}</span>
             </v-col>
           </v-row>
         </v-col>
         <v-col cols="3">
-          <v-btn v-if="!hasBusinessLookupChanges" text color="primary" id="bl-change-btn" @click="onChangeClicked()">
-            <v-icon small>mdi-pencil</v-icon>
-            <span>Change</span>
-          </v-btn>
```

Comment:
NAICS had the change functionality for use in Edit UI, but I think you're right: this component will NOT be used for changing a business anywhere -- only new lookups (or undo + new lookup).
👍